# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../clang/skip-distros.patch
# Copyright (C) 2019 - 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

There is no point wasting CPU cycles checking for other distributions.
We know we are "Unknown" aka T2.

  - Rene Rebe <rene@exactcode.de>

--- clang-10.0.0.src/lib/Driver/Distro.cpp.vanilla	2020-03-25 11:54:34.708521322 +0100
+++ clang-10.0.0.src/lib/Driver/Distro.cpp	2020-03-25 11:54:53.396521225 +0100
@@ -22,7 +22,7 @@
                                        const llvm::Triple &TargetOrHost) {
   // If we don't target Linux, no need to check the distro. This saves a few
   // OS calls.
-  if (!TargetOrHost.isOSLinux())
+  //if (!TargetOrHost.isOSLinux())
     return Distro::UnknownDistro;
 
   // If the host is not running Linux, and we're backed by a real file system,
